packages/core/admin/admin/src/pages/MarketplacePage/components/PluginCard/index.js

```diff
@@ -45,6 +45,50 @@ const PluginCard = ({ plugin, installedPluginNames, useYarn }) => {
     defaultMessage: 'Made by Strapi',
   });
 
+  // Decide to show install button or not
+  const showInstallButton = () => {
+    // Already installed
+    if (isInstalled) {
+      return (
+        <Box paddingLeft={4}>
+          <Icon as={Check} marginRight={2} width={12} height={12} color="success600" />
+          <Typography variant="omega" textColor="success600" fontWeight="bold">
+            {formatMessage({
+              id: 'admin.pages.MarketPlacePage.plugin.installed',
+              defaultMessage: 'Installed',
+            })}
+          </Typography>
+        </Box>
+      );
+    }
+
+    // In development, show install button
+    if (isInDevelopmentMode) {
+      return (
+        <CopyToClipboard
+          onCopy={() => {
+            trackUsage('willInstallPlugin');
+            toggleNotification({
+              type: 'success',
+              message: { id: 'admin.pages.MarketPlacePage.plugin.copy.success' },
+            });
+          }}
+          text={commandToCopy}
+        >
+          <Button size="S" startIcon={<Duplicate />} variant="secondary">
+            {formatMessage({
+              id: 'admin.pages.MarketPlacePage.plugin.copy',
+              defaultMessage: 'Copy install command',
+            })}
+          </Button>
+        </CopyToClipboard>
+      );
+    }
+
+    // Not in development and plugin not installed already. Show nothing
+    return null;
+  };
```

Comment:
What's the benefit of this vs moving it to its own component?

Reply:
Otherwise I would have had to write nested ternary operators. I thought it was more readable this way

Reply:
I think you could just copy paste this function to a React component. It looks like returning null is ok https://reactjs.org/docs/conditional-rendering.html#preventing-component-from-rendering. I don't know what is considered 'best practice' so I'll leave it up you to decide the best move.

Reply:
I think I misread your initial comment, I thought you were talking about this vs doing everything directly in the return. Another component is fine for me 👍